bootstrap.sh

```diff
@@ -22,6 +22,41 @@ function job_cleanup() {
     rm -fr $TMP_JOB_DIR
 }
 
+get_image_tag() {
+
+    local image_tag
+    local git_short_hash
+
+    git_short_hash=$(git rev-parse --short=7 HEAD)
+
+    if is_change_request; then
+        image_tag="pr-$(get_change_request_id)-${git_short_hash}"
+    else
+        image_tag="$git_short_hash"
+    fi
+
+    echo "$image_tag"
+}
+
+is_change_request() {
+  [ -n "$ghprbPullId" ] || [ -n "$gitlabMergeRequestId" ]
+}
+
+get_change_request_image_tag() {
```

Comment:
I don't see anywhere this gets used, and I also can't find the def for the above get_change_request_id, should these be the same?

Reply:
thanks that was a horrible typo, naming is hard and I keep changing names :facepalm:  - well spotted @wcmitchell !!